src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -14,76 +14,92 @@
 
 import java.util.Objects;
 
+// TODO: 11/26/16 this class is indicating there are two coordinate systems, one on the reference, the other on the assembled contig,
+//       the only asymmetry should be that the sequence have exact match to the contig, but has, in theory, in-exact alignment to the reference
+
 /**
  * A wrapper that wraps around the {@link AlnRgn} type returned by jBWA to represent one (out of the potentially multiple)
  * alignment record(s) of an locally-assembled contig.
  */
 @DefaultSerializer(AlignmentRegion.Serializer.class)
 class AlignmentRegion {
 
-    final String contigId;
+    static final String STRING_REP_SEPARATOR= "\t";
+    static final String DUMMY_ASM_ID = "ASSEMBLY";
```

Comment:
I'd at least make the dummy value be "NA" or something that indicates that `assemblyId` is not applicable when it's in use, then.